exec/execPull.sh

```diff
@@ -25,6 +25,9 @@ readonly REQKICK_REPO="https://github.com/Shippable/reqKick.git"
 readonly NODE_SHIPCTL_LOCATION="$NODE_SCRIPTS_LOC/shipctl"
 readonly NODE_ARCHITECTURE="x86_64"
 readonly NODE_OPERATING_SYSTEM="Ubuntu_14.04"
+# Note: There is a small chance that this URL might not exist if reports hasn't been tagged by the time the AMI starts.
```

Comment:
why dont we make this a dependency then?

Reply:
I'll add it as a dependency. I didn't add this initially as that job is likely to run much faster than it gets here, but better not to leave it to chance I guess.